app/components/auk/tab.hbs

```diff
@@ -0,0 +1,32 @@
+<Link
+  @route={{@route}}
+  @models={{this.models}}
+  @disabled={{@disabled}}
+  @current-when={{@current-when}}
+  @preventDefault={{@preventDefault}}
+  @query={{@query}}
+as |l|>
+  <li class="auk-tabs__tab {{if @isHierarchicalBack "auk-tabs__hierarchical-back"}} {{if l.isActive "auk-tabs__tab--active"}}" ...attributes>
```

Comment:
voor testen ga ik hier later testselectors aan toevoegen.
`data-test-auk-tab-hierarchical-back` voor deze tab
`data-test-auk-tab` voor normale tab
dan komt er vermoedelijk iets bij zoals
`{{if @isHierarchicalBack data-test-auk-tab-hierarchical-back data-test-auk-tab }}`

Kunnen hier later op terugkomen, mogelijk is het properder als ik dat in de .js file doe met `this.testTag`

Reply:
Inderdaad verder te bekijken. Omdat deze componenten weldra in een aparte addon zullen terechtkomen en (op langere termijn) zullen opgaan in appuniversum, denk ik dat het nuttig is om wat testen betreft nu al deze componenten te zien als code die buiten onze controle is, en waar je dus intern niet eenvoudig selectors aan kan toevoegen/wijzigen.